client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -148,14 +151,41 @@ const PluginDetailsCTA = ( props ) => {
 
 		return (
 			<div className="plugin-details-CTA__container">
-				<div className="plugin-details-CTA__installed-text">
-					{ translate( 'Installed and {{activation /}}', {
-						components: {
-							activation: active ? activeText : inactiveText,
-						},
-					} ) }
+				<div className="plugin-details-CTA__container-header">
+					<div className="plugin-details-CTA__installed-text">
+						{ translate( 'Installed and {{activation /}}', {
+							components: {
+								activation: active ? activeText : inactiveText,
+							},
+						} ) }
+					</div>
+					<div className="plugin-details-CTA__manage-plugin-menu">
+						<ManagePluginMenu plugin={ plugin } />
+					</div>
 				</div>
-				<Button>{ active ? translate( 'Deactivate' ) : translate( 'Activate' ) }</Button>
+
+				<ActivationButton plugin={ plugin } active={ active } />
+
+				<PluginAutoupdateToggle
+					site={ selectedSite }
+					plugin={ sitePlugin }
+					label={
+						<span className="plugin-details-CTA__autoupdate-text">
+							<span className="plugin-details-CTA__autoupdate-text-main">
+								{ translate( 'Enable autoupdate.' ) }
```

Comment:
```suggestion
								{ translate( 'Enable autoupdates.' ) }
```
just using plural here.

Reply:
Fixed at https://github.com/Automattic/wp-calypso/pull/66054/commits/ab174b95e0c024b43e1b6391826c270f1b33c70c